task7/main.js

```diff
@@ -0,0 +1,508 @@
+    hashTags: ['#world',  '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '14',
+    description: 'Более 76 тыс. человек во всем мире ',
+    createdAt: new Date('2020-03-20T23:00:00'),
+    author: 'Полина Лелис',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world', '#happiness', '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '15',
+    description: 'Хочу сверлить и днём, и ночью',
+    createdAt: new Date('2020-03-20T01:00:00'),
+    author: 'Соседка сверху',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world', '#happiness','#Exadel', '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '16',
+    description: 'Иванов',
+    createdAt: new Date('2017-03-18T23:00:00'),
+    author: 'Полина Лелис',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world', '#happiness','#Exadel', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '17',
+    description: 'Не забывайте надевать тёплые вещи!',
+    createdAt: new Date('2020-03-19T14:00:00'),
+    author: 'Весна',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world', '#happiness', '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '18',
+    description: 'Хочу быть Петровой',
+    createdAt: new Date('2020-06-19T03:00:00'),
+    author: 'Иванова Мария',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world',  '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '19',
+    description: 'спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+    createdAt: new Date('2030-03-14T22:00:00'),
+    author: 'Полина Лелис',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world', '#happiness', '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+    {
+
+    id: '20',
+    description: 'Любите себя сейчас',
+    createdAt: new Date('2001-03-15T03:00:00'),
+    author: 'Полина Лелис',
+    photoLink: 'https://www.pressball.by/images/stories/2020/03/20200310231542.jpg',
+    hashTags: ['#world',  '#love', '#BSU'],
+    likes: ['Лера Конфетка', 'Саша Привалов', 'Полина Филипович']
+    },
+
+  ];
+
+class PostList{
+
+  _postCollection = [];
+
+  constructor(postCollection) {
+    this._postCollection = postCollection;
+  }
+
+        static _validatePost(post){
```

Comment:
That the interesting moment and the diference between JS and Java private variables/methods
In Java private means none can access it except the current class.
In JS everything is public, so "private notation" appears a long ago.
But it has a difference (basically I'd say useful differences) in notation meaning
"private notation" means "don't touch it (except you know what you doing)" or "don't touch it is inner realization so you can corrupt something"
So the different meaning on notation produce different approach:
hide (means mark private) if it's something that is not helpful or dangerous form outside.

SO...
 _validatePost sounds potentially helpful and can not break something so it can be public.